tests/Unit/CommentTest.php

```diff
@@ -14,7 +14,7 @@ class CommentTest extends TestCase
     public function testPostedAt()
     {
         $comment = factory(Comment::class)->create();
-        $this->assertEquals($comment->posted_at->toDateTimeString(), now()->toDateTimeString());
+        $this->assertTrue($comment->posted_at->between(now()->subSeconds(2), now()));
```

Comment:
Nice catch,

Maybe we should use `assertEqualsWithDelta` with timestamps ?

With something like this:

```php
$this->assertEqualsWithDelta($user->registered_at->timestamp, now()->timestamp, 1);
```

Reply:
Thank you for your comments. I never knew about it before.
I fixed and pushed.